test/support/test_config.h

```diff
@@ -230,4 +230,7 @@
 #define _PSTL_RED_BY_SEG_WINDOWS_COMPILE_ORDER_BROKEN                                                                  \
     (_MSC_VER && TEST_DPCPP_BACKEND_PRESENT && __INTEL_LLVM_COMPILER <= 20250000)
 
+//oneAPI DPC+/C+ compiler produces 'Unexpected kernel lambda size issue' error

```

Comment:
oneAPI DPC+/C+ compiler (https://github.com/intel/llvm?tab=readme-ov-file#oneapi-dpc-compiler) does not define `__INTEL_LLVM_COMPILER`. The fix below applies only to Intel® oneAPI DPC++/C++ Compiler.

Likely, the issue is present with both compiler, but I do not insist on expanding it right now, because other "broken" macro from this file do not do it.

Reply:
```suggestion
// Intel(R) oneAPI DPC++/C++ compiler produces 'Unexpected kernel lambda size issue' error

```